hikari/api/rest.py

```diff
@@ -3865,7 +3865,8 @@ def fetch_members(
             itself will not raise anything.
 
         !!! warning
-            This endpoint requires the `GUILD_MEMBERS` intent. Alternatively,
+            This endpoint requires the `GUILD_MEMBERS` intent to be enabled in
```

Comment:
What does "authenticated with it" mean in this context when authenticating with an intent just doesn't exist on rnst?

Reply:
Updated